localization/english/dvg_flavor_venice_l_english.yml

```diff
@@ -0,0 +1,50 @@
+venice.1.f:0 "To build a city where it is impossible to build a city is madness in itself, but to build there one of the most elegant and grandest of cities is the madness of genius. - Alexander Herzen"
+venice_background.1.a:0 "I want to know more of Venice's history."
+venice_background.1.b:0 "I want to know more about Venice's possessions."
+venice_background.1.c:0 "I want to know more about the Patrician's and Venice's political situation."
+venice_background.1.d:0 "Desponsamus te, mare, in signum veri perpetuique dominii."
+#Now the call for Greek independence can be heard far and wide throughout the Greek possessions, while to the West, nationalistic forces are plotting to unify Italy, once and for all. Yet in the face of these threats, the Patrician nobility continues to resist the winds of change, fearful that any change to the status quo will end their vicelike grip over power. As a Doge, you will have to guide Venice during this perilous century or watch as Venice fades into ever greater irrelevance.
+
+#Venice State Trait
+state_trait_venice_arsenal:0 "Venetian Arsenal"
+state_trait_venice_arsenal_desc:0 "Constructed around 1104, the Venetian Arsenal has been one of the largest state-owned shipyards and armories in the world, producing the majority of all Venetian ships. As long as Venice controls it's Arsenal, it is certain that Venice will continue to rule the waves."
+
+#Ideology Trait
+ideology_venician_patrician:0 "Venetian Patricians"
+ideology_venician_patrician_desc:0 "The Venetian patricians are an aristocratic class of nobles who hold the right to vote within the Republic of Venice. The heriditary nature of the title, alongside the continued political exclusion of all other parts of Venician society from goverment, has made the patricians reluctant to undertake reforms or give up their power."
+
+#Interest Group Descriptions
+ig_barnabotti:0 "Barnabotti"
+ig_barnabotti_desc:0 "A particular class of patricians whose families, though having fallen on hard financial times, still held the right to vote in the Great Council. 
+The Bernabotti defacto represent the voice of the poor, though this voice is rarely heard as the Bernabotti are far more concerned with the selling of their votes so as to regain their former status. #italic Interestingly, the term Barnabotti comes from the zone that many of these families lived and congregated, Campo San Barnaba, which was distant from the city center and thus cheaper to rent."
```

Comment:
Not sure if the #italic Interestingly part is causing issues? For some reason not seeing desc of these ig's in game.